membrane_curvature/base.py

```diff
@@ -0,0 +1,126 @@
+"""
+MembraneCurvature
+=======================================
+
+:Author: Estefania Barreto-Ojeda
+:Year: 2021
+:Copyright: GNU Public License v3
+
+MembraneCurvature calculates the mean and Gaussian curvature of
+surfaces derived from a selection of reference.
+"""
+
+import numpy as np
+import warnings
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
+
+
+class MembraneCurvature(AnalysisBase):
+
+    r"""
+    MembraneCurvature is a tool to calculate membrane curvature.
+
+    Parameters
+    ----------
+    universe : Universe or AtomGroup
+        An MDAnalysis Universe object.
+    select : str or iterable of str, optional. Default: 'all'
```

Comment:
The syntax for default values seems to be according to  [numpy docstring standard](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) (which is what we follow  [Userguide: Working with the docs](https://userguide.mdanalysis.org/2.0.0-dev0/contributing_code.html#working-with-the-code-documentation))

```suggestion
    select : str or iterable of str, optional, default: 'all'
```

but you can leave out the value because it's already part of the function signature.